### PR TITLE
Shorted text. To fix the overlays.

### PR DESCRIPTION
**Pull request by yaroslav4167**
GUI without this fix:
jP1i

### CODE REVIEW

File: src/main/resources/assets/refinedstorage/lang/ru_ru.lang [hunk 1]
```diff
@@ -7,11 +7,11 @@ key.refinedstorage.clearGridCraftingMatrix=Очистить сетку созд
 gui.refinedstorage:controller.0=Контроллер
 gui.refinedstorage:controller.1=Творческий контроллер
 gui.refinedstorage:grid=Терминал
-gui.refinedstorage:grid.craft=Создание
-gui.refinedstorage:crafting_grid=Терминал создания
-gui.refinedstorage:pattern_grid=Терминал шаблонов
+gui.refinedstorage:grid.craft=+
```
Comment (CrossVas) [inline src/main/resources/assets/refinedstorage/lang/ru_ru.lang]: It's a good idea

**Comment by CrossVas** [comment]
It's not a good way to fix this problem.
Instead of making the text shorter, it's better to make Search bar to be 4 slots long (if this is possible)

**Comment by yaroslav4167** [comment]
Shorten terminal name displays only in terminals GUI.
I think it is not difficult to distinguish the terminals inside.

**Comment by raoulvdberge** [comment]
I think it's a fine fix.

**Comment by raoulvdberge** [comment]
I will merge it later.

**Pull request merged**